gRPC.podspec

```diff
@@ -0,0 +1,186 @@
+                               "#{objc_client_root}/GRPCCall+Tests.h",
+                               "#{objc_client_root}/GRPCCall+ChannelArg.h"
+      ss.private_header_files = "#{objc_client_root}/private/GRPCCore/*.h"
+      ss.source_files = "#{objc_client_root}/private/GRPCCore/*.{h,m}",
+                        "#{objc_client_root}/GRPCCall+ChannelArg.h",
+                        "#{objc_client_root}/GRPCCall+ChannelArg.m",
+                        "#{objc_client_root}/GRPCCall+ChannelCredentials.h",
+                        "#{objc_client_root}/GRPCCall+ChannelCredentials.m",
+                        "#{objc_client_root}/GRPCCall+Cronet.h",
+                        "#{objc_client_root}/GRPCCall+Cronet.m",
+                        "#{objc_client_root}/GRPCCall+OAuth2.h",
+                        "#{objc_client_root}/GRPCCall+OAuth2.m",
+                        "#{objc_client_root}/GRPCCall+Tests.h",
+                        "#{objc_client_root}/GRPCCall+Tests.m",
+                        "#{objc_client_root}/GRPCCallLegacy.m"
+  
+      # Certificates, to be able to establish TLS connections:
+      ss.resource_bundles = { 'gRPCCertificates' => ['etc/roots.pem'] }
+  
+      ss.dependency "#{s.name}/Interface-Legacy", version
+      ss.dependency "#{s.name}/Interface", version
+      ss.dependency 'gRPC-Core', version
+      ss.dependency 'gRPC-RxLibrary', version
+  
+      ss.ios.deployment_target = '9.0'
+      ss.osx.deployment_target = '10.10'
+      ss.tvos.deployment_target = '10.0'
+      ss.watchos.deployment_target = '4.0'
+    end
+  
+    s.subspec 'GRPCCoreCronet' do |ss|
+      ss.header_mappings_dir = objc_client_root
+  
+      ss.source_files = "#{objc_client_root}/GRPCCall+Cronet.h",
+                        "#{objc_client_root}/GRPCCall+Cronet.m",
+                        "#{objc_client_root}/private/GRPCCore/GRPCCoreCronet/*.{h,m}"
+      ss.dependency "#{s.name}/GRPCCore", version
+      ss.dependency 'gRPC-Core/Cronet-Implementation', version
+      ss.dependency 'CronetFramework'
+  
+      ss.ios.deployment_target = '9.0'
+    end
+  
+    # CFStream is now default. Leaving this subspec only for compatibility purpose.
+    s.subspec 'CFStream' do |ss|
+      ss.dependency "#{s.name}/GRPCCore", version
+  
+      ss.ios.deployment_target = '9.0'
+      ss.osx.deployment_target = '10.10'
+      ss.tvos.deployment_target = '10.0'
+      ss.watchos.deployment_target = '4.0'
+    end
+  
+    s.subspec 'InternalTesting' do |ss|
+      ss.dependency "#{s.name}/GRPCCore", version
+      ss.public_header_files = "#{objc_client_root}/internal_testing/*.h"
+      ss.source_files = "#{objc_client_root}/internal_testing/*.{h,m}"
+      ss.header_mappings_dir = objc_client_root
+  
+      ss.ios.deployment_target = '9.0'
+      ss.osx.deployment_target = '10.10'
+      ss.tvos.deployment_target = '10.0'
+      ss.watchos.deployment_target = '4.0'
+    end
+  end
+  
```

Comment:
so odd that you could have an extra line but no newline at the end 😂

Reply:
fixed